### PR TITLE
No no no no no no... WAIT WAIT WAIT WAIT

### PR DESCRIPTION
**Pull request by RigglePrime**
:^)

### CODE REVIEW

**Comment by Imaginos16** [comment]
I literally woke up to this

https://user-images.githubusercontent.com/77556824/186482851-f283e39a-95de-43e4-a6fe-5b035f4a2a3f.MP4

File: code/modules/mob/living/carbon/human/species_types/humans.dm [hunk 1]
```diff
@@ -17,10 +22,36 @@
...
+		// Force everyone to have at least fox features. I am evil and fucked up. We don't want to
+		// override cat ears, so I'll do it like this. If you don't like it sue me.
+		if(H.dna.features["tail_cat"] == "None")
+			H.dna.features["tail_cat"] = "Fox"
+		if(H.dna.features["ears"] == "None")
+			H.dna.features["ears"] = "Fox"
+
+		if(H.dna.features["ears"] == "Fox")
+			var/obj/item/organ/internal/ears/fox/ears = new
+			ears.Insert(H, drop_if_replaced = FALSE)
```
Comment (Time-Green) [inline code/modules/mob/living/carbon/human/species_types/humans.dm:47]: This is awful, we have an organ generation framework for this shit

Comment (RigglePrime) [reply code/modules/mob/living/carbon/human/species_types/humans.dm:47]: I had just about 10 minutes to remove merge conflicts before the next round started, actual good code comes later

**Comment by Vanadiom** [comment]
It worked well I suppose (Why did you do this D: )

**Comment by RigglePrime** [comment]
The voices in my head told me to

**Pull request closed without merging**